app/src/main/res/navigation/nav_graph.xml

```diff
@@ -35,27 +35,15 @@
         android:name="com.bottlerocketstudios.brarchitecture.ui.home.HomeFragment"
         android:label="@string/home_title"
         tools:layout="@layout/home_fragment">
-        <action
-            android:id="@+id/action_homeFragment_to_repositoryFragment"
-            app:destination="@id/repositoryFragment" />
         <action
             android:id="@+id/action_homeFragment_to_userFragment"
             app:destination="@id/userFragment" />
         <action
             android:id="@+id/action_homeFragment_to_snippetsFragment"
             app:destination="@id/snippetsFragment" />
-    </fragment>
-    <fragment
-        android:id="@+id/repositoryFragment"
-        android:name="com.bottlerocketstudios.brarchitecture.ui.repository.RepositoryFragment"
-        android:label="@string/empty_label"
-        tools:layout="@layout/repository_fragment">
         <action
-            android:id="@+id/action_repositoryFragment_to_repositoryFileFragment"
-            app:destination="@id/repositoryFileFragment" />
-        <action
-            android:id="@+id/action_repositoryFragment_to_repositoryFolderFragment"
-            app:destination="@id/repositoryFolderFragment" />
+            android:id="@+id/actionnHomeToRepositoryBrowserFragment"
```

Comment:
@ianslinker-br nit: Refactor to `actionHomeToRepositoryBrowserFragment` (including usage in `HomeFragment`)